site/content/en/docs/Guides/Client SDKs/unity.md

```diff
@@ -226,3 +226,27 @@ The properties for the Unity Agones SDK can be found in the Inspector.
   - Whether the server sends a health ping to the Agones sidecar. (default: `true`)
 - Log Enabled
   - Debug Logging Enabled. Debug logging for development of this Plugin. (default: `false`)
+
+## Contributing
+Thank you for your interest in contributing to the Agones Unity client library. Setting up a local workspace to help develop the Agones Game Server Unity Client SDK involves cloning the Agones source code, having a host unity project, and allow the unity project to run the external tests in the Unity test runner.
+
+### A copy of Agones source code
+If you don't already have a local copy of the Agones source code you can either (1) clone the official [googleforgames/agones repository](https://github.com/googleforgames/agones) for quicker access or (2) create a fork then clone your fork. Make note of where you clone the project to for the next setup steps!
+
+### Setting up the Host Unity Project
+You will want a unity project to "host" a local copy of the agones unity sdk source code. We like the unity project name "Agones Unity SDK Host Project" but name the host unity project whatever you like. Open the project so unity can complete its initial project setup.
```

Comment:
Nit:
```suggestion
You will want a Unity project to "host" a local copy of the Agones Unity SDK source code. We like the Unity project name "Agones Unity SDK Host Project" but name the host Unity project whatever you like. Open the project so Unity can complete its initial project setup.
```

Same comment for the below sections as well. Agones, Unity, and SDK should be capitalized.